Gemfile

```diff
@@ -51,6 +50,7 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'rubocop', '~> 0.59.2', require: false
```

Comment:
Rubocop should be in the test  group also, this way  travis can run it, it's weird it can now 🤔

Reply:
Done